test/dynamo/test_exc.py

```diff
@@ -183,6 +183,11 @@ def fn001(x):
   Developer debug context: Called `torch._dynamo.graph_break()` with args `[]`, kwargs `{}`
 
 User code traceback:
+  File "test_exc.py", line N, in test_graph_break_log
+    torch.compile(fn001, backend="eager")(torch.randn(1))
+
+========== `torch.compile` tracing started here ==========
+
```

Comment:
Can this be "collapsed" somehow? I would have expected something like:
```
  File "test_exc.py", line N, in test_graph_break_log
    torch.compile(fn001, backend="eager")(torch.randn(1))
  File "test_exc.py", line N, in fn001
    return fn002(x)
  File "test_exc.py", line N, in fn002
    torch._dynamo.graph_break()
```

Reply:
Forgot to update this test aha